ssr-opengraph/src/utils.ts

```diff
@@ -58,15 +58,26 @@ export function formatPrice(price: string) {
 
 export async function getProperties(nft: NFT) {
   if (!nft.meta) {
-    const response = await fetch(ipfsToCdn(nft.metadata));
-    const data = (await response.json()) as NFTMeta;
-
-    return {
-      name: data.name,
-      description: data.description,
-      title: `${data.name} | Low Carbon NFTs`,
-      cdn: ipfsToCdn(data.image),
-    };
+    try {
+      const response = await fetch(ipfsToCdn(nft.metadata));
+      const data = (await response.json()) as NFTMeta;
+
+      return {
+        name: data.name,
+        description: data.description,
+        title: `${data.name} | Low Carbon NFTs`,
+        cdn: ipfsToCdn(data.image),
+      };
+    } catch (error) {
+      console.log('Error', error);
+
+      return {
+        name: nft.name,
+        description: '',
+        title: `${nft.name} | Low Carbon NFTs`,
```

Comment:
```suggestion
        name: nft?.name,
        description: '',
        title: `${nft?.name} | Low Carbon NFTs`,
```

If some error happens, `nft` may be empty.